make-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+cd `dirname $0`
+export NEW_VERSION="$1"
+export ONLY_RUN_ON_BRANCH="2016-03-10-mf-make-release-script"
+
+test -z "$NEW_VERSION" \
+    && ( echo "Please provide a version number to move to."; exit 1 )
+git status -b | grep -q "On branch $ONLY_RUN_ON_BRANCH" \
+    || ( echo "Please only run this script on master."; exit 1 )
+
+perl -i -pe 's/(version:\s*).*/${1}'"$NEW_VERSION"'/' package.yaml
```

Comment:
I don't speak perl. @Mikolaj could you take a look, maybe you have better knowledge about the topic. :)